scalafix/tests/src/test/scala/migrate/RuleSuite.scala

```diff
@@ -1,13 +1,60 @@
 package migrate
 
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileWriter
+
+import scala.meta.XtensionTokenizeInputLike
+import scala.meta.io.AbsolutePath
+
 import org.scalatest.FunSuiteLike
+import scalafix.internal.patch.PatchInternals
 import scalafix.testkit._
 
 class RuleSuite extends AbstractSemanticRuleSuite with FunSuiteLike {
   val (passing, failing) = testsToRun.partition(!_.path.testName.contains("_fails"))
   passing.foreach(runOn)
-//  runOnSpecificTest("Rename")
+//  runSpecificTests("Cats2")
+
   //   for running only one test if using Intellij
-  def runOnSpecificTest(name: String): Unit =
-    testsToRun.filter(_.path.testName.toLowerCase.contains(name.toLowerCase())).foreach(runOn)
+  private def runSpecificTests(name: String): Unit =
```

Comment:
Just to be able to ovewrite test result in /scalafix-output

Reply:
It's nice to have!